examples/complete/config/ue2-testing.yaml

```
@@ -0,0 +1,16 @@
+projects:
+  globals:
+    stage: testing
+    environment: ue2
+
+  terraform:
+    example2:
+      workspace_enabled: true
+      terraform_version: 0.13.4
+      auto_apply: true
+      filename_prefix: "*.*"
```

```suggestion
      filename_trigger: "*.*"
```

https://github.com/cloudposse/terraform-tfe-cloud-infrastructure-automation/pull/2/files#diff-187b92b2a3650f0c7ae6047f2d51a582b19dbaec7f87ebde30fa519687e348a7R66